### ISSUE TITLE
Feature Req: Craft Plugin Store: Add "Project" or "Client" fields to Purchase Confirmation Step

### ISSUE THREAD

**Issue by wooge** [pixelandtonic/craftnet#177]
On the Craft Plugin Store, on the billing confirmation page when making a purchase, could we have a field or two for internal project tracking? Like "Project" or "Client"? They can be simple plain text inputs.

When purchasing plugins and Craft Pro/Commerce licenses, having the ability to mark the invoice/receipt with a client or project is useful for accounting. This would help agencies like mine keep better tabs on who's paying for what ;)

Thank you!!

**Comment by lauriechu** [comment]
Yes, please! We are also an agency with clients who use many of the same plugins. We use our Craft ID but, _after the purchase_, I have to go in to Licenses/Plugins and click on the License Key then edit to add notes. When I receive the Receipt(s) via email, there is no reference info so I have to login again to confirm I'm billing the client for the correct plugin(s) and providing them with the correct invoice.

Many steps just to match client to purchase. Thanks!

**Comment by lauriechu** [comment]
Another year gone by and plugin renewals, so just dredging this issue up again. Maybe a text field in 2022?
